README.rst

```diff
@@ -142,6 +142,10 @@ bitbucket
 branch
   Which branch to track? Default is the repository's default.
 
+use_max_tag
+  Set this to ``true`` to check for the max tag on BitBucket. Will return the biggest one
+  sorted by ``pkg_resources.parse_version``.
```

Comment:
I tried to use `LooseVersion` of distutils, but it will raise errors when comparing `v1.1` and `1.2`. `pkg_resources.parse_version` doesn't have the problem and could compare regardless the `v` prefix. Also this is what `pip` uses currently.